sdk/monitor/azure-mgmt-monitor/dev_requirements.txt

```diff
@@ -3,9 +3,9 @@ aiohttp>=3.0; python_version >= '3.5'
 -e ../../../tools/azure-devtools
 -e ../../storage/azure-mgmt-storage
 -e ../../eventhub/azure-mgmt-eventhub
--e ../../loganalytics/azure-mgmt-loganalytics
+azure-mgmt-loganalytics
```

Comment:
Curious why we need live azure-mgmt-storage, azure-mgmt-eventhub, azure-mgmt-web, azure-mgmt-network, azure-mgmt-applicationinsights?

Reply:
i'm unsure about the azure-mgmt-monitor package - I can spend some time later to figure it out